kolibri_explore_plugin/views.py

```diff
@@ -144,10 +177,37 @@ def get(self, request):
             json.dumps(jobs_response), content_type="application/json"
         )
 
+    def import_job(self, task):
+        content_fallback = paths.get_content_fallback_paths()
+        if content_fallback:
+            task.update({"type": "DISKIMPORT"})
+            # Just getting the first one
+            directory = os.path.dirname(content_fallback[0])
```

Comment:
Yes, getting the first one would work. At least in the Android app the content fallback path will be set only if there is an external device with a KOLIBRI_DATA folder and the corresponding folder structure.